src/main/resources/assets/additions/lang/en_us.lang

```diff
@@ -23,7 +23,6 @@ commands.additions.throw.success=%1$s successfully threw %2$s
 commands.additions.move.usage=/additions.move <global | relative | facing | toward>
 commands.additions.move.global.usage=/additions.move global <x speed> <y speed> <z speed>
 commands.additions.move.facing.usage=/additions.move relative <horizontal speed forward/back> [speed y] [horizontal speed right/left]
-commands.additions.move.facing.usage=/additions.move facing <speed forward/back> [speed up/down] [speed right/left]
```

Comment:
Oof yeah... That's a copypasta error. This line's fine actually. The line above it should be commands.additions.move.relative.usage

Looks like I'm never actually using the relative usage in the command though, I'll have to fix that too.